src/sass/vic-v2.scss

```diff
@@ -216,9 +216,11 @@ input[type=range] {
   }
 }
 
-.cropper-control-label-container {
-  margin: 10px;
-  min-width: 106px;
+.cropper-control-row {
+  .cropper-control-label-container {
```

Comment:
This was being overridden on the review page which was removing the space between the buttons. Using a more specific query fixed it.